internal/testrunner/reporters/console.go

```diff
@@ -0,0 +1,40 @@
+package reporters
+
+import (
+	"fmt"
+
+	"github.com/jedib0t/go-pretty/table"
+
+	"github.com/elastic/elastic-package/internal/testrunner"
+)
+
+func init() {
+	testrunner.RegisterReporter(ConsoleReporter, ReportConsole)
+}
+
+const (
+	// ConsoleReporter reports test results in a console-friendly tabular format
+	ConsoleReporter testrunner.TestReporter = "console"
+)
+
+// ReportConsole returns the given test results formatted in a console-friendly tabular format
+func ReportConsole(results []testrunner.TestResult) (string, error) {
+	t := table.NewWriter()
+	t.AppendHeader(table.Row{"Package", "Data stream", "Test name", "Result", "Time taken"})
```

Comment:
Is the "time taken" enforced by some xUnit rules/convention? Maybe "time elapsed"?

Reply:
Yeah, it's an attribute in the xUnit test case element (`<testcase time="...">`). I believe it's more convention than rule but that's where it comes from. I'm happy to change the name of the header in the Console reporter to "time elapsed".